### ISSUE TITLE
Could not complete installing plugin

### ISSUE THREAD

**Issue by dyson-yamashita** [fossamagna/amplify-category-console-notification#7]
**Describe the bug**
I followed the Installation and executed the commands, but the following error occurred and I could not complete the configuration.

```
Cannot find module '/home/ec2-user/.nvm/versions/node/v12.22.0/lib/node_modules/amplify-category-console-notification/lib/commands/add'
Require stack:
- /home/ec2-user/.nvm/versions/node/v12.22.0/lib/node_modules/amplify-category-console-notification/lib/index.js
- /home/ec2-user/.nvm/versions/node/v12.22.0/lib/node_modules/@aws-amplify/cli/lib/execution-manager.js
- /home/ec2-user/.nvm/versions/node/v12.22.0/lib/node_modules/@aws-amplify/cli/lib/index.js
- /home/ec2-user/.nvm/versions/node/v12.22.0/lib/node_modules/@aws-amplify/cli/bin/amplify
```

I did not see any commands under lib.
I would like to use this plugin, so if there is a way to deal with this, could you please let me know?

```
$ ls /home/ec2-user/.nvm/versions/node/v12.22.0/lib/node_modules/amplify-category-console-notification/lib/
index.js
$ 
```

**To Reproduce**
1. `$ npm install -g amplify-category-console-notification`
2. `$ amplify plugin scan`
3. `$ amplify console-notification add`

**Expected behavior**
Ability to complete the plugin configuration.

**Screenshots**
<img width="807" alt="スクリーンショット 2021-06-02 6 48 23" src="https://user-images.githubusercontent.com/4074527/120394578-99dabc00-c36e-11eb-8058-16735f7e8229.png">

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: none
 - Version: Amazon Linux AMI release 2018.03

**Smartphone (please complete the following information):**
 - Device: none
 - OS: none
 - Browser none
 - Version none

**Additional context**

**Comment by fossamagna** [comment]
@dyson-yamashita Thank you. Because of there is not `commands/add.js` in published package, you see the error. I will fix this issue as soon as I can.

**Comment by fossamagna** [comment]
@dyson-yamashita I released v0.2.2. Please try it.
